### ISSUE TITLE
Is it possible to provide src files to fork library? 

### ISSUE THREAD

**Issue by AlexanderMoroz** [jfeltkamp/cookiesjsr#6]
Hi! First of all thanks for the great application! 

We really like functionality of your app and how easy it use with Drupal, but we need to have possibility to change layout of front-end part. Would be great to be create your own fork, but there is no src files... 

If you can somehow let us use that would be great

Thanks in advance!

**Comment by JPustkuchen** [comment]
That would be absolutely great and could give the library a push and further supporters / devs! :) 
Talked to the maintainer about this in the past, he'll check if that's possible due to Sponsor agreement.

**Comment by AlexanderMoroz** [comment]
Thanks! Sounds promising :)

**Comment by shumer** [comment]
That sounds like a great idea, this will make the application much more flexible and will allow us to use it more widely!

**Comment by AlexanderMoroz** [comment]
> That would be absolutely great and could give the library a push and further supporters / devs! :)
> Talked to the maintainer about this in the past, he'll check if that's possible due to Sponsor agreement.


Sorry to bother you, but need to know. Any updates on this?

**Comment by geek-merlin** [comment]
Just found this after i created a similar one on d.o: [Clarify Licensing of JS [#3251511] | Drupal.org](https://www.drupal.org/project/cookies/issues/3251511)

So note that even if the source is provided or un-obfuscated, releasing any changed JS is forbidden by current license.

**Comment by JPustkuchen** [comment]
Hi Axel,

you're right and we already discused this with @jfeltkamp. He wanted to talk to the client he developed this for, if I remember right, if he'd be so kind to make this open source. I guess we'd all appreciate that a lot.

**Comment by JPustkuchen** [comment]
@jfeltkamp Any news from the sponsoring company, if they might do the Open Source world a favor and release the source code under an open source license?
That would also allow providing pull requests and better help to fix bugs and making improvements.

Thanks :)

**Comment by finnlewis** [comment]
Me too! I would love to be able to contribute to improving this but am currently blocked by the lack of source and the licence. For the most part, it works great, but there is always room for improvement, happy to help when we can.
Thanks for all the work so far though @jfeltkamp 👍

**Comment by superbiche** [comment]
I don't like adding "+1"-ish comments, but it would be really helpful indeed. I started working on a patch to make a WYSIWYG `bannerText` but unfortunately the React part breaks. It's a requirement for my client and I really don't want to go back to similar modules as none of these are as good as this one. :crossed_fingers: for this to happen. In the meantime thanks a lot for the quality work already done and released publicly :clap: .

**Comment by jhuhta** [comment]
This is a showstopper for us in using this library and the related Drupal module, despite them otherwise being quite nicely built and feature rich. Some customers can't rely on something that doesn't have the source opened - also, we would need to be able to customize some parts, like the markup on the cookie banner, which obviously isn't possible as it's provided by the React app.

**Comment by lexsoft00** [comment]
+1 any news about this?

**Comment by steveoriol** [comment]
+1 any news about this?
I need to render the `bannerText` as HTML... 🙏

**Comment by vredeling** [comment]
@jfeltkamp any updates on changing the license and opening the source?

**Comment by finnlewis** [comment]
@jfeltkamp this would be really cool to have if at all possible.

**Comment by JPustkuchen** [comment]
@ALL here: I think we won't see this changing. So if the community wants to solve this, https://github.com/jfeltkamp/cookiesjsr-vue needs to be pushed forward so we can simply remove this dependency and use the open source vue alternative.

@jfeltkamp already implemented a lot over there!

**Comment by JPustkuchen** [comment]
Fixed in 2.x, but unsolvable in 1.x. I think we can close this @jfeltkamp ?

**Comment by jfeltkamp** [comment]
Please use 2.x, source now available.
